config/crd/bases/hnc.x-k8s.io_hierarchyconfigurations.yaml-e

```diff
@@ -0,0 +1,103 @@
+
```

Comment:
ditto why there's a `-e`?

Reply:
Yeah, new computer and didn't have kustomize installed, so the makefile used the 'edit' binary (instead of kustomize edit) that created those funny enough. Removed and added to gitignore.